File: LorePdks.COMMON/DTO/Auth/Securty/Ekran/EkranDTO.cs
using LorePdks.COMMON.DTO.Base;
using System;
using System.Collections.Generic;

namespace LorePdks.COMMON.DTO.Auth.Securty.Ekran
{
    public class EkranDTO : BaseDTO
    {
        public string ekranAdi { get; set; }
        public string ekranYolu { get; set; }
        public string ekranKodu { get; set; }
        public string aciklama { get; set; }
        public int? ustEkranId { get; set; }
        public int siraNo { get; set; }
        public string ikon { get; set; }
        public bool aktif { get; set; }
        public List<EkranDTO> altEkranlar { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/EkranDTO.ts
import e from "express";
import { baseDto } from "./baseDto";

/**
 * Kullanıcı bilgisini temsil eden model
 */
export interface EkranDTO extends baseDto {
 
  ekranAdi: string;
  ekranYolu: string;
  ekranKodu: string;
  aciklama: string;
  ustEkranId?: number;
  siraNo?: number;
  ikon: string;
  aktif: boolean;
  altEkranlar: EkranDTO[];
}

